.github/workflows/ci.yml

```diff
@@ -122,7 +122,7 @@ jobs:
           image: paritytech/srtool
           chain: ${{ matrix.runtime }}
           # Can remove that when we can build with stable, see: https://github.com/integritee-network/parachain/issues/200
-          tag: 1.66.1
+          tag: 1.71.0
```

Comment:
This is here for the following reason: By default the newest srtool image is taken. However all images after 1.66.1 only have the stable toolchain installed, which fails as long as we have the nightly in our `rust-toolchain.toml`.

It was substrate that prevented the change to a stable rust-toolchain if I remember correctly, so we can just try switching the toolchain to some stable version, and maybe it works.

Reply:
it does. thx